src/components/ActionCard.tsx

```diff
@@ -76,7 +79,7 @@ const ActionCard = ({
       >
         <Image
           src={image}
-          width={220}
+          width={imageWidth}
```

Comment:
Seeing an image height bug now:

<img width="1775" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/aaa1e3dd-07b8-4b03-ab39-85c01838524e">


Canonical:
<img width="1767" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/f615f4c4-b923-43c4-b20a-adad5d6ffd36">

```suggestion
          width={imageWidth}
          maxH="full"
```

@pettinarip  This does the trick, but does that seem like an appropriate approach to you?

Reply:
Nice catch! that solution looks perfect